worlds/khrecom/__init__.py

```diff
@@ -0,0 +1,170 @@
+                "World Card Monstro",
+                "World Card Atlantica",
+                "World Card Halloween Town",
+                "World Card Neverland",
+                "World Card Hollow Bastion",
+                "World Card 100 Acre Wood",
+                "World Card Twilight Town",
+                "World Card Destiny Islands"]
+            starting_worlds = self.random.sample(possible_starting_worlds, min(self.options.starting_worlds.value, len(possible_starting_worlds)))
+            for starting_world in starting_worlds:
+                self.multiworld.push_precollected(self.create_item(starting_world))
+        if self.options.starting_cure:
+            self.multiworld.push_precollected(self.create_item("Card Set Cure"))
+        
+        item_pool: List[KHRECOMItem] = []
+        total_locations = len(self.multiworld.get_unfilled_locations(self.player))
+        sleights_added = 0
+        enemy_cards_added = 0
+        shuffled_item_list = list(item_table.items())
+        self.random.shuffle(shuffled_item_list)
+        for name, data in shuffled_item_list:
+            # Ignore filler, it will be added in a later stage.
+            if data.category not in ["World Unlocks", "Gold Map Cards", "Friend Cards", "Enemy Cards", "Sleights"]:
+                continue
+            elif data.category == "Enemy Cards" and enemy_cards_added < self.options.enemy_card_amount and name not in ["Enemy Card " + card_name for card_name in self.options.exclude_enemy_cards.value]:
+                enemy_cards_added = enemy_cards_added + 1
+                item_pool.append(self.create_item(name))
+            elif data.category == "Sleights" and sleights_added < self.options.sleight_amount and name not in ["Sleight " + sleight_name for sleight_name in self.options.exclude_sleights.value]:
+                sleights_added = sleights_added + 1
+                item_pool.append(self.create_item(name))
+            elif data.category not in ["Sleights", "Enemy Cards"] and name not in starting_worlds and name != "Friend Card Pluto":
+                item_pool.append(self.create_item(name))
+        
+        # Fill any empty locations with filler items.
+        while len(item_pool) < total_locations:
+            item_pool.append(self.create_item(self.get_filler_item_name()))
+        
+        self.multiworld.itempool += item_pool
+
+    def place_predetermined_items(self) -> None:
+        self.get_location("12F Exit Hall Larxene II (Enemy Cards Larxene)").place_locked_item(self.create_item("Friend Card Pluto"))
+        self.get_location("Final Marluxia").place_locked_item(self.create_item("Victory"))
+
+    def get_filler_item_name(self) -> str:
+        fillers = {}
+        exclude = []
+        for card in self.options.exclude_cards.value:
+            exclude.append("Card Set " + card)
+        fillers.update(get_items_by_category("Sets", exclude))
+        return self.random.choices([filler for filler in fillers.keys()], k=1)[0]
```

Comment:
Why not just `self.random.choice`? Choices is really for returning more than one from a collection, while you're only returning one.